test/orbits/keplerian.jl

```diff
@@ -0,0 +1,467 @@
+            aR_star = sky_coords["a"][i],
+            P = sky_coords["period"][i],
+            incl = sky_coords["incl"][i],
+            t0 = sky_coords["t0"][i],
+            ecc = sky_coords["e"][i],
+            Omega = 0.0,
+            omega = sky_coords["omega"][i],
+        )
+        for i in 1:length(sky_coords["t0"])
+    ]
+
+    # Compute coords
+    t = sky_coords["t"]
+    x = Matrix{Float64}(undef, length(sky_coords["t"]), length(sky_coords["t0"]))
+    y = similar(x)
+    z = similar(x)
+    for (col, orbit) in enumerate(orbits)
+        for (row, _t) in enumerate(t)
+            pos = relative_position(orbit, _t)
+            x[row, col] = pos[1]
+            y[row, col] = pos[2]
+            z[row, col] = pos[3]
+        end
+    end
+
+    # Compare
+    m = sky_coords["m"]
+    r = hypot.(x, y)
+    r_Transits = r[m]
+    r_batman = sky_coords["r_batman"][m]
+
+    @test sum(m) > 0
+    @test allclose(r_Transits, r_batman, atol=2e-5)
+    @test all(>(0), z[m])
+    no_transit = @. (z[!(m)] < 0) | (r[!(m)] > 2)
+    @test all(no_transit)
+end
+
+@testset "KeplerianOrbit: construction performance" begin
+    b_rho_star = @benchmark KeplerianOrbit(
+        rho_star = 2.0,
+        R_star = 0.5,
+        period = 2.0,
+        ecc = 0.0,
+        t0 = 0.0,
+        incl = π / 2.0,
+        Omega = 0.0,
+        omega = 0.0,
+    )
+
+    b_rho_star_units = @benchmark KeplerianOrbit(
+        rho_star = 2.0u"g/cm^3",
+        R_star = 0.5u"Rsun",
+        period = 2.0u"d",
+        ecc = 0.0,
+        t0 = 0.0u"d",
+        incl = 90.0u"°",
+        Omega = 0.0u"°",
+        omega = 0.0u"°",
+    )
+
+    b_aR_star = @benchmark KeplerianOrbit(
+        aR_star = 7.5,
+        P = 2.0,
+        incl = π / 2.0,
+        t0 = 0.0,
+        ecc = 0.0,
+        Omega = 0.0,
+        omega = 0.0,
+    )
+
+    b_aR_star_units = @benchmark KeplerianOrbit(
+        aR_star = 7.5,
+        P = 2.0u"d",
+        incl = 90.0u"°",
+        t0 = 0.0u"d",
+        ecc = 0.0,
+        Omega = 0.0u"°",
+        omega = 0.0u"°",
+    )
+
+    # Units
+    @test median(b_rho_star_units.times) ≤ 100_000 # ns
+    @test b_rho_star_units.allocs ≤ 500
+    @test median(b_aR_star_units.times) ≤ 100_000   # ns
+    @test b_aR_star_units.allocs ≤ 500
+
+    if Base.VERSION ≥ v"1.6"
+        @test b_rho_star.allocs == b_rho_star.memory == 0
+        @test median(b_rho_star.times) ≤ 500 # ns
+        @test b_aR_star.allocs == b_aR_star.memory == 0
+        @test median(b_aR_star.times) ≤ 500 # ns
+    else
+        # TODO: investigate performance regression
+        @test median(b_rho_star.times) ≤ 20_000 # ns
+        @test median(b_aR_star.times) ≤ 20_000 # ns
+    end
+end
+
+@testset "KeplerianOrbit: orbital elements" begin
+    orbit = KeplerianOrbit(
+        cos_omega=√(2)/2, sin_omega=√(2)/2,
+        period=2.0, t0=0.0, b=0.01, M_star=1.0, R_star=1.0, ecc=0.0,
+    )
+    @test orbit.omega == atan(orbit.sin_omega/orbit.cos_omega)
```

Comment:
```suggestion
    @test orbit.omega == atan(orbit.sin_omega, orbit.cos_omega)
```
unless you explicitly don't want to restrict from -pi to pi

Reply:
nope, just forgot about that variant!